gnucash/import-export/aqb/gschemas/org.gnucash.dialogs.flicker.gschema.xml.in

```diff
@@ -0,0 +1,24 @@
+<schemalist gettext-domain="@PROJECT_NAME@">
+
+  <schema id="org.gnucash.dialogs.flicker" path="/org/gnucash/dialogs/flicker/">
+    <key name="window-size" type="(ii)">
+      <default>(-1,-1)</default>
+      <summary>Last window size</summary>
+      <description>This setting describes the size of the window when it was last closed.
+        The numbers are the width and height of the window. </description>
+    </key>
```

Comment:
Change this for my state file changes...

  <schema id="org.gnucash.dialogs.flicker" path="/org/gnucash/dialogs/flicker/">
    <key name="last-geometry" type="(iiii)">
      <default>(-1,-1,-1,-1)</default>
      <summary>Last window position and size</summary>
      <description>This setting describes the size and position of the window when it was last closed.
        The numbers are the X and Y coordinates of the top left corner of the window
        followed by the width and height of the window.</description>
    </key>